EduID/Flows/VerifyIdentity/VerifyAlreadyUsedViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  VerifyAlreadyUsedViewController.swift
+//  eduID
+//
+//  Created by Yasser Farahi on 03/04/2025.
+//
+
+import UIKit
+
+final class VerifyAlreadyUsedViewController: BaseViewController {
+    
+    var email: String
+    
+    init(email: String) {
+        self.email = email
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError( "init(coder:) has not been implemented" )
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        screenType = .verifyAlreadyUsedScreen
+    }
+    
+    
+    private func setupUI() {
+        view.subviews.forEach { $0.removeFromSuperview() }
+        
+        let mainTitle: UILabel = UILabel.posterTextLabelBicolor(
+            text: L.EppnAlreadyLinked.Title.VerificationFailed.localization,
+            size: 24,
+            primary: L.EppnAlreadyLinked.Title.VerificationFailed.localization
+        )
+        let description = UILabel.subtitleLabel(text: L.EppnAlreadyLinked.InfoExternalAccountWithEmail(args: "\n\(email)").localization)
```

Comment:
Email can be nillable. For that case we have a different error message: `InfoExternalAccountWithoutEmail`